ci_framework/roles/cifmw_cephadm/tasks/bootstrap.yml

```diff
@@ -98,6 +98,18 @@
         {{ cifmw_cephadm_container_ns + '/' + cifmw_cephadm_container_image + ':' + cifmw_cephadm_container_tag|string }}
       changed_when: false
 
+    - name: Switch HA container images if a custom container registry is used
```

Comment:
The cephadm role already has a tasks file for doing these config sets. Please see:

https://github.com/openstack-k8s-operators/ci-framework/blob/reproducer/va1-layout/ci_framework/roles/cifmw_cephadm/tasks/cephadm_config_set.yml

tripleo-ansible already has code you can borrow in cephadm_config_set.yaml to set `mgr/cephadm/container_image_keepalived` and `mgr/cephadm/container_image_haproxy`.

https://github.com/openstack/tripleo-ansible/blob/master/tripleo_ansible/roles/tripleo_cephadm/tasks/cephadm_config_set.yaml#L87-L104

Could you use something like the above in `ci_framework/roles/cifmw_cephadm/tasks/cephadm_config_set.yml` so that the tasks for config setting is more consistent?

Reply:
Yeah, it makes sense to add the task mentioned by John in [cephadm_config_set.yaml](https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/cifmw_cephadm/tasks/cephadm_config_set.yml): this allows to not have to `redeploy` rgw (or any other component that requires the ingress daemon) because you should have the right images in place before applying the spec via the `orchestrator`.